src/sentry/conf/server.py

```diff
@@ -1609,3 +1603,22 @@ def get_sentry_sdk_config():
         'topic': KAFKA_EVENTS,
     },
 }
+
+
+SOUTH_TESTS_MIGRATE = os.environ.get('SOUTH_TESTS_MIGRATE', '0') == '1'
+
+SOUTH_MIGRATION_CONVERSIONS = (
```

Comment:
this was the easiest way i found to do this that wouldn't leave you in an unknown state

e.g. this will fail very very hard if you haven't run the required south migrations before running django migrations

Reply:
I think we can add extra safety steps here. So if the user has a south migration history but hasn't run any of these south migrations we can just error and tell them to upgrade to Sentry 10 first.

Reply:
Nit: wouldn't it be better to make this a dict instead of having this comment above explaining the fields? The reason I'm asking is because that comment would not exist on custom config files if this is ever used in custom config files. If that is very unlikely, then I think this structure is okay.

Reply:
This is unlikely to be used in custom config files. Mostly I'm just keeping this structure because it was already in place, if I extend it any further I might switch it over to be a dict.